src/main/java/core/basesyntax/model/dto/FruitDataDto.java

```diff
@@ -0,0 +1,25 @@
+package core.basesyntax.model.dto;
+
+public class FruitDataDto {
+    private String operationType;
+    private String fruitName;
```

Comment:
Також, у тебе є модель Fruit, давай зберігати дані про фрукт так:
```suggestion
    private Fruit fruit;
```

Reply:
done